docker/docker_image.go

```diff
@@ -39,25 +41,70 @@ func (i *Image) SourceRefFullName() string {
 	return i.src.ref.ref.Name()
 }
 
-// GetRepositoryTags list all tags available in the repository. Note that this has no connection with the tag(s) used for this specific image, if any.
+// GetRepositoryTags list all tags available in the repository. Note that this
+// has minimal connection with the tag(s) used for this specific
+// ImageReference; the tag will be used to determine policy-based access (see
+// PolicyConfigurationIdentity), but will otherwise be ignored. (This is a
+// backward-compatible shim method which calls the module-level GetRepositoryTags)
 func (i *Image) GetRepositoryTags(ctx context.Context) ([]string, error) {
```

Comment:
(I wouldn’t mind dropping this entirely, but that would require updating `skopeo` and be more work. Keeping this shim is perfectly fine.)